docs/user/03-15-kubeconfig-endpoint.md

```diff
@@ -0,0 +1,64 @@
+# Kubeconfig Endpoint
+
+## Overview
+
+The Kyma Environment Broker (KEB) provides an endpoint that lets you retrieve the kubeconfig file for accessing an SAP BTP, Kyma runtime.
+
+## HTTP Request
+
+```
+GET /kubeconfig/{instance_id}
+```
+No request body is required.
+
+## Response Structure
+
+The endpoint returns a standard Kubernetes kubeconfig file.
+- [**Cluster Name**](04-05-cluster-name.md) is used as the cluster name, context name, and—when OIDC authentication is configured the user name.
+- If no OIDC configuration is present, the users section is omitted entirely.
+- When multiple OIDC configurations are provided, user entries are generated using an incremental naming convention:
+  - The first user is named **CLUSTER_NAME**
+  - The second user is named **CLUSTER_NAME-2**
+  - The third user is named **CLUSTER_NAME-3**
+  - The pattern continues accordingly for subsequent users.
+
+### Response Body
+
```

Comment:
Again, how about adding something like:
"You get a response similar to the following."